packages/frontend/src/utils/wallet.ts

```diff
@@ -1685,7 +1731,7 @@ export default class Wallet {
 
     async getPublicKey(accountId = this.accountId) {
         const account = await this.getAccount(accountId);
-        const signer = account.inMemorySigner || account.connection.signer;
+        const signer = account.signerIgnoringLedger || account.connection.signer;
```

Comment:
Over here too.

Reply:
exactly the same thing, as what have I mentioned above. I am totally fed up with this file, I gonna rewrite this whole file when i free.

But temporarily, I try to make it working first. This refactoring is working.